Makefile.in

```diff
@@ -62,7 +62,12 @@ install: all
 distcheck:
 	@$(ECHO) @PACKAGE_VERSION@
 
-dist: config.h.in configure
-	tar -s +^+msktutil-@PACKAGE_VERSION@/+ -jcvf msktutil-@PACKAGE_VERSION@.tar.bz2 \
+tar: config.h.in configure
+	tar -s +^+msktutil-@PACKAGE_VERSION@/+ -cvf msktutil-@PACKAGE_VERSION@.tar \
```

Comment:
If we go this route and keep the tar archive as an intermediate file, we can avoid the phony target and just use the filename directly, ie.

```
@PACKAGE_NAME@-@PACKAGE_VERSION@.tar: config.h.in configure
	tar -s +^+@PACKAGE_NAME@-@PACKAGE_VERSION@/+ -cvf $@ $(SOURCE) $(HEADERS) (...)
```

Unrelated to the actual change, but I wonder whether we can improve on hardcoding the `-s` option which is specific to BSD `tar`. (With GNU `tar` we could use `tar --transform s+^+@PACKAGE_NAME@-@PACKAGE_VERSION@/+ (...)` instead.) I'll see to come up with an additional cleanup.